src/hydra/parseHydraDocumentation.js

```diff
@@ -91,6 +91,40 @@ function removeTrailingSlash(url) {
 }
 
 /**
+ *
+ * @param {object} docs
+ * @param {object} property
+ * @return {string|null}
+ */
+function findRelatedClass(docs, property) {
+  // Use the entrypoint property's owl:equivalentClass if available
+  if (property['http://www.w3.org/2000/01/rdf-schema#range']) {
+    for (const range of property['http://www.w3.org/2000/01/rdf-schema#range']) {
+      const equivalentClass = range['http://www.w3.org/2002/07/owl#equivalentClass'];
+      if (equivalentClass && 'http://www.w3.org/ns/hydra/core#member' === equivalentClass[0]['http://www.w3.org/2002/07/owl#onProperty'][0]['@id']) {
+        //console.log('==>'+equivalentClass[0]['http://www.w3.org/2002/07/owl#allValuesFrom'][0]['@id']);
```

Comment:
you kept the `console.log`s?

Reply:
I noticed, I'm working on a follow up PR.